Python/Product/PythonTools/PythonTools/LanguageServerClient/PylanceLanguageServer.cs

```diff
@@ -65,17 +65,33 @@ internal sealed class PylanceLanguageServer {
                 FileName = nodePath,
                 WorkingDirectory = serverFolderPath,
                 RedirectStandardInput = true,
+                RedirectStandardError = true,
                 RedirectStandardOutput = true,
                 UseShellExecute = false,
                 CreateNoWindow = true,
-                Arguments = $"{debugArgs} \"{serverFilePath}\" -- --stdio --cancellationReceive=file:{this.CancellationFolderName}",
+                Arguments = $"{debugArgs} \"{serverFilePath}\" -- --stdio --cancellationReceive=file:{this.CancellationFolderName} --verbose",
```

Comment:
Oh I should probably remove the --verbose. Or only add it when debugging

Reply:
yup good call